src/scanomf.d

```diff
@@ -187,7 +187,8 @@ void scanOmfObjModule(void delegate(const(char)* name, int pickAny) pAddSymbol,
                 parseName(&p, name.ptr);
                 p += (recTyp == PUBDEF) ? 2 : 4; // skip offset
                 parseIdx(&p); // skip type index
-                pAddSymbol(name.ptr, 0);
+                const len = strlen(name.ptr);
+                pAddSymbol(name[0 .. len], 0);
```

Comment:
>When the reviewer sees a name being introduced, she starts to worry about the use of that name later in the function.

In fact I should add I did - this is not a threoretical. The first rvalue use I understood instantly, and then I saw the name being introduced. I started looking down the function (minding the `static if`s) thinking there must be a reason for it.